scripts/RunUpdate.py

```diff
@@ -128,16 +131,16 @@ def updatepyenv(self, dev_build):
                 for repo in list_of_repo:
                     command = 'git clone --recursive git@github.com:wwPDB/{}.git'.format(repo.rstrip())
                     self.__exec(command, working_directory=source_dir)
-                    command = 'pip install --edit {}/ -c {}'.format(repo, constraintfile)
+                    command = 'pip install {} --edit {}'.format(pip_extra_urls, repo)
                     self.__exec(command, working_directory=source_dir)
         else:
-            reqfile = self.__cparser.get('DEFAULT', 'piprequirements')
-            command = 'pip install -U --extra-index-url {} --trusted-host {} --extra-index-url https://pypi.anaconda.org/OpenEye/simple -r {} -c {}'.format(urlpath, urlreq.netloc, reqfile, constraintfile)
+            # reqfile = self.__cparser.get('DEFAULT', 'piprequirements')
```

Comment:
Why is reqfile commented out here?  For a non-dev build looks like it will fail for update.

Reply:
it isn't - it moved to line 119 outside of the else.